INFO:uvicorn.error:Started server process [19920]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Started server process [14184]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
DEBUG:multipart.multipart:Calling on_part_begin with no data
DEBUG:multipart.multipart:Calling on_header_field with data[42:61]
DEBUG:multipart.multipart:Calling on_header_value with data[63:122]
DEBUG:multipart.multipart:Calling on_header_end with no data
DEBUG:multipart.multipart:Calling on_header_field with data[124:136]
DEBUG:multipart.multipart:Calling on_header_value with data[138:162]
DEBUG:multipart.multipart:Calling on_header_end with no data
DEBUG:multipart.multipart:Calling on_headers_finished with no data
DEBUG:multipart.multipart:Calling on_part_data with data[166:1704]
DEBUG:multipart.multipart:Calling on_part_data with data[0:2]
DEBUG:multipart.multipart:Calling on_part_end with no data
DEBUG:multipart.multipart:Calling on_end with no data
DEBUG:multipart.multipart:Calling on_part_begin with no data
DEBUG:multipart.multipart:Calling on_header_field with data[42:61]
DEBUG:multipart.multipart:Calling on_header_value with data[63:122]
DEBUG:multipart.multipart:Calling on_header_end with no data
DEBUG:multipart.multipart:Calling on_header_field with data[124:136]
DEBUG:multipart.multipart:Calling on_header_value with data[138:162]
DEBUG:multipart.multipart:Calling on_header_end with no data
DEBUG:multipart.multipart:Calling on_headers_finished with no data
DEBUG:multipart.multipart:Calling on_part_data with data[166:1704]
DEBUG:multipart.multipart:Calling on_part_data with data[0:2]
DEBUG:multipart.multipart:Calling on_part_end with no data
DEBUG:multipart.multipart:Calling on_end with no data
INFO:uvicorn.error:Started server process [3872]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Started server process [12912]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
DEBUG:multipart.multipart:Calling on_part_begin with no data
DEBUG:multipart.multipart:Calling on_header_field with data[42:61]
DEBUG:multipart.multipart:Calling on_header_value with data[63:122]
DEBUG:multipart.multipart:Calling on_header_end with no data
DEBUG:multipart.multipart:Calling on_header_field with data[124:136]
DEBUG:multipart.multipart:Calling on_header_value with data[138:162]
DEBUG:multipart.multipart:Calling on_header_end with no data
DEBUG:multipart.multipart:Calling on_headers_finished with no data
DEBUG:multipart.multipart:Calling on_part_data with data[166:1704]
DEBUG:multipart.multipart:Calling on_part_data with data[0:2]
DEBUG:multipart.multipart:Calling on_part_end with no data
DEBUG:multipart.multipart:Calling on_end with no data
INFO:uvicorn.error:Started server process [22296]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
DEBUG:multipart.multipart:Calling on_part_begin with no data
DEBUG:multipart.multipart:Calling on_header_field with data[42:61]
DEBUG:multipart.multipart:Calling on_header_value with data[63:122]
DEBUG:multipart.multipart:Calling on_header_end with no data
DEBUG:multipart.multipart:Calling on_header_field with data[124:136]
DEBUG:multipart.multipart:Calling on_header_value with data[138:162]
DEBUG:multipart.multipart:Calling on_header_end with no data
DEBUG:multipart.multipart:Calling on_headers_finished with no data
DEBUG:multipart.multipart:Calling on_part_data with data[166:1704]
DEBUG:multipart.multipart:Calling on_part_data with data[0:2]
DEBUG:multipart.multipart:Calling on_part_end with no data
DEBUG:multipart.multipart:Calling on_end with no data
ERROR:uvicorn.error:Exception in ASGI application
Traceback (most recent call last):
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\uvicorn\protocols\http\h11_impl.py", line 373, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\uvicorn\middleware\proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\fastapi\applications.py", line 208, in __call__
    await super().__call__(scope, receive, send)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\middleware\errors.py", line 181, in __call__
    raise exc from None
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\middleware\errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\exceptions.py", line 82, in __call__
    raise exc from None
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\routing.py", line 580, in __call__
    await route.handle(scope, receive, send)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\routing.py", line 241, in handle
    await self.app(scope, receive, send)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\routing.py", line 52, in app
    response = await func(request)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\fastapi\routing.py", line 226, in app
    raw_response = await run_endpoint_function(
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\fastapi\routing.py", line 159, in run_endpoint_function
    return await dependant.call(**values)
  File "D:\User\jkhair\IdeaProjects\Certifai\MLCapstone\app\.\main.py", line 84, in create_upload_file
    return templates.TemplateResponse("home_copy.html", {"request": request, "df": df})
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\templating.py", line 80, in TemplateResponse
    template = self.get_template(name)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\templating.py", line 67, in get_template
    return self.env.get_template(name)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\jinja2\environment.py", line 997, in get_template
    return self._load_template(name, globals)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\jinja2\environment.py", line 958, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\jinja2\loaders.py", line 125, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\jinja2\loaders.py", line 214, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: home_copy.html
INFO:uvicorn.error:Started server process [17512]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
INFO:uvicorn.error:Started server process [23868]
INFO:uvicorn.error:Waiting for application startup.
INFO:uvicorn.error:Application startup complete.
DEBUG:multipart.multipart:Calling on_part_begin with no data
DEBUG:multipart.multipart:Calling on_header_field with data[42:61]
DEBUG:multipart.multipart:Calling on_header_value with data[63:122]
DEBUG:multipart.multipart:Calling on_header_end with no data
DEBUG:multipart.multipart:Calling on_header_field with data[124:136]
DEBUG:multipart.multipart:Calling on_header_value with data[138:162]
DEBUG:multipart.multipart:Calling on_header_end with no data
DEBUG:multipart.multipart:Calling on_headers_finished with no data
DEBUG:multipart.multipart:Calling on_part_data with data[166:1704]
DEBUG:multipart.multipart:Calling on_part_data with data[0:2]
DEBUG:multipart.multipart:Calling on_part_end with no data
DEBUG:multipart.multipart:Calling on_end with no data
ERROR:uvicorn.error:Exception in ASGI application
Traceback (most recent call last):
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\uvicorn\protocols\http\h11_impl.py", line 373, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\uvicorn\middleware\proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\fastapi\applications.py", line 208, in __call__
    await super().__call__(scope, receive, send)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\middleware\errors.py", line 181, in __call__
    raise exc from None
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\middleware\errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\exceptions.py", line 82, in __call__
    raise exc from None
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\routing.py", line 580, in __call__
    await route.handle(scope, receive, send)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\routing.py", line 241, in handle
    await self.app(scope, receive, send)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\routing.py", line 52, in app
    response = await func(request)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\fastapi\routing.py", line 226, in app
    raw_response = await run_endpoint_function(
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\fastapi\routing.py", line 159, in run_endpoint_function
    return await dependant.call(**values)
  File "D:\User\jkhair\IdeaProjects\Certifai\MLCapstone\app\.\main.py", line 84, in create_upload_file
    return templates.TemplateResponse("home_copy.html", {"request": request, "df": df})
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\templating.py", line 80, in TemplateResponse
    template = self.get_template(name)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\starlette\templating.py", line 67, in get_template
    return self.env.get_template(name)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\jinja2\environment.py", line 997, in get_template
    return self._load_template(name, globals)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\jinja2\environment.py", line 958, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\jinja2\loaders.py", line 125, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\Users\jkhai\miniconda3\envs\AIShip_conda\lib\site-packages\jinja2\loaders.py", line 214, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: home_copy.html
